low-level/framework/base/sspl_constants.py

```diff
@@ -143,7 +143,8 @@ class RaidDataConfig(Enum):
     MISMATCH_COUNT_RESPONSE = '0'
     RAID_RESULT_DIR = "/tmp"
     RAID_RESULT_FILE_PATH = "/tmp/result_raid_health_file"
-    MAX_RETRIES = 50
+    MAX_RETRIES = 10
```

Comment:
do we need 10 retries? is 5 or even 3 sufficient?

Reply:
As per the observation, the RAID device file takes few seconds to change its state.
Keeping 3 or 5 retries would be less.